datasets/laroseda/README.md

```diff
@@ -0,0 +1,172 @@
+
+## Dataset Creation
+
+### Curation Rationale
+
+The samples are preprocessed in order to eliminate named entities. This is required to prevent classifiers from taking the decision based on features that are not related to the topics. 
+For example, named entities that refer to politicians or football players names can provide clues about the topic. For more details, please read the [paper](https://arxiv.org/abs/1901.06543).
+
+### Source Data
+
+
+#### Data Collection and Normalization
+
+For the data collection, one of the largest Romanian e-commerce platform was targetted. Along with the textual content of each review, the associated star ratings was also collected in order to automatically assign labels to
+the collected text samples. 
+
+
+#### Who are the source language producers?
+
+The original text comes from one of the largest e-commerce platforms in Romania.
+
+### Annotations
+
+#### Annotation process
+
+As mentioned above, LaRoSeDa is composed of product reviews from one of the largest e-commerce websites in Romania. The resulting samples are automatically tagged with the star rating assigned by the users.
+
+#### Who are the annotators?
+
+N/A
+
+### Personal and Sensitive Information
+
+The textual data collected for LaRoSeDa consists in product reviews freely available on the Internet. 
+To the best of authors' knowledge, there is no personal or sensitive information that needed to be considered in the said textual inputs collected.
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+This dataset is part of an effort to encourage text classification research in languages other than English. Such work increases the accessibility of natural language technology to more regions and cultures. 
+In the past three years there was a growing interest for studying Romanian from a Computational Linguistics perspective. However, we are far from having enough datasets and resources in this particular language.
+
+### Discussion of Biases
+
+*We note that most of the negative reviews (5,561) are rated with one star. Similarly, most of the positive reviews (6,238) are rated with five stars. Hence, the corpus is highly polarized.*
+
+### Other Known Limitations
+
+*The star rating might not always reflect the polarity of the text. We thus acknowledge that the automatic labeling process is not optimal, i.e. some labels might be noisy.*
+
+## Additional Information
+
+### Dataset Curators
+
+Published and managed by Anca Tache, Mihaela Gaman and Radu Tudor Ionescu.
+
+### Licensing Information
+
+CC BY-SA 4.0 License
+
+### Citation Information
+
+@article{
+    tache2101clustering,
+    title={Clustering Word Embeddings with Self-Organizing Maps. Application on LaRoSeDa -- A Large Romanian Sentiment Data Set},
+    author={Anca Maria Tache and Mihaela Gaman and Radu Tudor Ionescu},
+    journal={ArXiv},
+    year = {2021}
+}
```

Comment:
````suggestion
```
@article{
    tache2101clustering,
    title={Clustering Word Embeddings with Self-Organizing Maps. Application on LaRoSeDa -- A Large Romanian Sentiment Data Set},
    author={Anca Maria Tache and Mihaela Gaman and Radu Tudor Ionescu},
    journal={ArXiv},
    year = {2021}
}
```
````
add code block